arduino/src/main.cpp

```diff
@@ -36,7 +36,7 @@ void loop() {
     HTTPClient http;
     http.begin(serverAddress);
     http.addHeader("Content-Type", "application/json");
-     int httpResponseCode = http.sendRequest("GET", payload);
+     int httpResponseCode = http.sendRequest("POst", payload);

```

Comment:
Chyba niechcący pushnąłeś zmiany w kodzie. Tutaj powinien być tylko plik requirements. A tak btw to masz tutaj literówkę